modules/serverless-build-func-kn.adoc

```diff
@@ -13,7 +9,7 @@ Before you can run a function, you must build the function project by using the
 name: example-function
 namespace: default
 runtime: node
-image: <image_from_registry>
+image: <image_from_repository>
```

Comment:
This should remain "registry". The only command where the term changed is the `create` command.
```suggestion
image: <image_from_registry>
```